Dell/BIOS/PSAppDeploy_Dell_E7450_(SCCM_SAMPLE).md

```diff
@@ -0,0 +1,22 @@
+#SCCM Detection Logic (PowerShell)
+```
+#Set minimum bios version
+$BiosMinimum = "A10"
+
+#Get BIOS config
+$Bios = Get-WmiObject Win32_Bios
+
+#Check for minimum version
+If($Bios.SMBIOSBIOSVersion -ge $BiosMinimum)
+{
+    #requirements met, write something to output
+    Write-Host "BIOS is current"
+}
+#requirements not met, return nothing
+```
+
+#SCCM Return Code (Dell BIOS Returns are non-standard)
+Value: 2
```

Comment:
This doesn't render nicely: https://github.com/bciu22/PowerShell-Modules/blob/DellBIOS/Dell/BIOS/PSAppDeploy_Dell_E7450_(SCCM_SAMPLE).md